includes/ECommerce.php

```diff
@@ -675,4 +675,19 @@ function check_url_match( $brand_name, $site_url ) {
 			update_option( 'showMigrationSteps', false );
 		}
 	}
+
+	/**
+	 * Suppress WP-Form Notice using css
+	 *
+	 * @return void
+	 */
+
+	public function hide_wp_pointer_with_css() {
+		$brand = $this->container->plugin()->id;
+		if ( 'bluehost' === $brand || 'hostgator' === $brand ) {
```

Comment:
Don't we need this for `crazy-domains`

Reply:
Updated. Applicable for Bluehost, Hostgator and crazy domains.